### ISSUE TITLE
The explanation of activeDeadlineSeconds is inaccurate and misleading

### ISSUE THREAD

**Issue by s-kawamura-w664** [kubernetes/website#28825]
**This is a Bug Report**

**Problem:**

The [init-containers documentation](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#detailed-behavior) includes:

Use `activeDeadlineSeconds` on the Pod and `livenessProbe` on the container to
prevent init containers from failing forever. The active deadline includes init
containers.

This explanation is inaccurate and misleading.
The activeDeadlineSeconds can be used to interrupt init container. However, it also affects the container after the initContainers are successful.
The combination of the Pod's activeDeadlineSeconds and the container's livenessProbe is an effective means when the container is terminated like batch, though it cannot be used in a container that continue to process.
In other words, the activeDeadlineSeconds are inconsistent in case of restartPolicy=Always.

**Proposed Solution:**

Update the explanation as following.

Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever. The active deadline includes init containers.  
The pod will be killed even if the restartPolicy on the pod is Always after the period specified in activeDeadlineSeconds.
Therefore, activeDeadlineSeconds should only be used when container will complete like batch.

**Page to Update:**

https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

**Additional Information:**
/kind bug
/language en
/assign

**Comment by RA489** [comment]
PR to fix the issue https://github.com/kubernetes/website/pull/28363

**Comment by s-kawamura-w664** [comment]
/close
I close this issue because PR#28363 was fixed.
